lib/Doctrine/ORM/Proxy/ProxyFactory.php

```diff
@@ -175,7 +164,6 @@ private function createCloner(ClassMetadata $classMetadata, EntityPersister $ent
                     continue;
                 }
 
-                $property->setAccessible(true);
```

Comment:
Is this unnecessary code or a decision that only public properties should be used?

Reply:
This method is a no-op since PHP 8.1.